src/internal/connector/graph_connector_helper_test.go

```diff
@@ -294,6 +293,39 @@ func checkMessage(
 	assert.Equal(t, expected.GetUniqueBody(), got.GetUniqueBody(), "UniqueBody")
 }
 
+// checkFlags is a helper function to check equality of models.FollowupFlabables
+// OdataTypes are omitted as these do change between msgraph-sdk-go versions
+func checkFlags(
+	t *testing.T,
+	expected, got models.FollowupFlagable,
+) {
+	assert.Equal(t, expected.GetCompletedDateTime(), got.GetCompletedDateTime())
+	assert.Equal(t, expected.GetDueDateTime(), got.GetDueDateTime())
+	assert.Equal(t, expected.GetFlagStatus(), got.GetFlagStatus())
+	assert.Equal(t, expected.GetCompletedDateTime(), got.GetCompletedDateTime())
+	assert.Equal(t, expected.GetCompletedDateTime(), got.GetCompletedDateTime())
```

Comment:
should these be some other value? This field is checked 3x (once at start and 2x here)

Reply:
Resolved within [8fc5ec0](https://github.com/alcionai/corso/pull/1715/commits/8fc5ec0874d54b6db9e7c19e282ef243d64c8ff8)